src/components/ImageView/RegionView/RegionViewComponent.tsx

```diff
@@ -430,10 +431,10 @@ export class RegionViewComponent extends React.Component<RegionViewComponentProp
             if (!isAstBadPoint(wcsPos)) {
                 const dist = frame.distanceMeasuring;
                 if (!dist.isCreating && !dist.showCurve) {
-                    dist.start = imagePos;
+                    dist.setStart(imagePos.x, imagePos.y);
                     dist.setIsCreating(true);
                 } else if (dist.isCreating) {
-                    dist.finish = imagePos;
+                    dist.setFinish(imagePos.x, imagePos.y);
                     dist.updateTransformedPos(frame.spatialTransform);
                     dist.setIsCreating(false);
                 } else {
```

Comment:
also change line 442 to `setStart`

Reply:
The region dialog uses <td> to wrap the start/end instead of label and labelInfo props. Do you mind if I change back to how it was before?

Reply:
I think following the code style of region dialogs works.